### ISSUE TITLE
Doesn't work with Apple Silicon M1 😥

### ISSUE THREAD

**Issue by josephschito** [Mr0grog/editorconfig-textmate#41]

**Comment by Mr0grog** [comment]
Hey @josephschito, I’m sorry for the slow response here. Unfortunately I don’t have an M1 Mac to test with and have been extra-busy with some COVID-related projects lately, but am more than happy to take any PRs to help fix the build (or other issues!).

**Comment by josephschito** [comment]
No prob, I'm trying to build it in my machine (with M1 chip) but I got this error:

```
Building EditorConfig-Core...
CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error: The source directory "/Users/joseph/Downloads/editorconfig-textmate-master/lib/editorconfig-core" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
/Users/joseph/Library/Developer/Xcode/DerivedData/editorconfig-textmate-acjzzwhpulylgvfowxxzivdrpamn/Build/Intermediates.noindex/editorconfig-textmate.build/Debug/EditorConfig-Core.build/Script-CB60467916F6B558004753CD.sh: line 6: /Users/joseph/Downloads/editorconfig-textmate-master/lib/editorconfig-core: is a directory
make: *** No targets specified and no makefile found.  Stop.
cp: lib/libeditorconfig_static.a: No such file or directory
cp: /Users/joseph/Downloads/editorconfig-textmate-master/lib/editorconfig-core/include/editorconfig/*: No such file or directory
cp: /Users/joseph/Downloads/editorconfig-textmate-master/lib/editorconfig-core/LICENSE: No such file or directory
Command PhaseScriptExecution failed with a nonzero exit code
```

**Comment by Mr0grog** [comment]
`editorconfig-core` is included as a submodule — did you `git submodule init`?

**Comment by josephschito** [comment]
Thank you @Mr0grog, it built! 😃 
But it doesn't work yet...

I noticed these 3 warnings ⚠️ 

`source/additions/NSView+EditorConfig.m:34:54: Instance method '-representedFilename' not found (return type defaults to 'id')`

`source/additions/NSView+EditorConfig.m:81:28: 'accessibilityAttributeValue:' is deprecated: first deprecated in macOS 10.10 - Use the NSAccessibility protocol methods instead (see NSAccessibilityProtocols.h)`

`source/additions/NSView+EditorConfig.m:149:15: 'accessibilitySetValue:forAttribute:' is deprecated: first deprecated in macOS 10.10 - Use the NSAccessibility protocol methods instead (see NSAccessibilityProtocols.h)`

**Comment by Mr0grog** [comment]
I should hopefully have this building as a universal bundle later this week.

**Comment by Mr0grog** [comment]
A test release of this is available as v0.5.0-alpha1: https://github.com/Mr0grog/editorconfig-textmate/releases/tag/v0.5.0-alpha1

If you try it out, please let me know if you run into any issues.

**Comment by josephschito** [comment]
@Mr0grog It works!!! 👏👏👏
Thank you! 🙏
